CHANGELOG.md

```diff
@@ -20,6 +20,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - OneDrive backups no longer include a user's non-default drives.
 - OneDrive and SharePoint file downloads will properly redirect from 3xx responses.
 
+ ### Known Issues
+ - SharePoint document library data can't be restored after the library has been deleted.
```

Comment:
maybe `if the library has been deleted`? After kind of makes it sound like it will definitely happen

Reply:
Sorry - late to review. Should add this will be addressed once support to restore to a different library than source is added.